src/utils/helpers.ts

```diff
@@ -40,3 +40,45 @@ export function isValidData(data) {
   }
   return true
 }
+
+export const phoneLinks = (data) => {
+  // Change phone to tap to dial.
+  const replacePattern =
+    /\(?(\d{3})\)?[- ]?(\d{3}-\d{4})(?!([^<]*>)|(((?!<a).)*<\/a>))/g
+  if (data) {
+    return data.replace(
+      replacePattern,
+      '<a target="_blank" href="tel:$1-$2">$1-$2</a>'
+    )
+  }
+  return data
+}
+
+export const drupalToVaPath = (content) => {
+  let replaced = content
+  if (content) {
+    replaced = content.replace(/href="(.*?)(png|jpg|jpeg|svg|gif)"/g, (img) =>
+      img
+        .replace('http://va-gov-cms.lndo.site/sites/default/files', '/img')
+        .replace('http://dev.cms.va.gov/sites/default/files', '/img')
+        .replace('http://staging.cms.va.gov/sites/default/files', '/img')
+        .replace('http://prod.cms.va.gov/sites/default/files', '/img')
+        .replace('https://prod.cms.va.gov/sites/default/files', '/img')
+        .replace('http://cms.va.gov/sites/default/files', '/img')
+        .replace('https://cms.va.gov/sites/default/files', '/img')
```

Comment:
mebbe use the regex `https?:\/\/(test\.)?(dev.|staging.|prod.)?cms\.va\.gov\/sites\/default\/files`, since that'll accomodate our test environments too.

Reply:
ooh nice! thanks Nate

Reply:
I'm not very good at regex, how would I add  `va-gov-cms.ddev.site` to this?

would it be something like or is it not possible:

/https?:\/\/(test\.)?(dev.|staging.|prod.|va-gov-cms.)?ddev\?site\?cms\.va\.gov\/sites\/default\/files/

```
.replace(
/https?:\/\/(test\.)?(dev.|staging.|prod.)?cms\.va\.gov\/sites\/default\/files/,
'/files'
)
.replace('http://va-gov-cms.ddev.site/sites/default/files', '/files')
```

Reply:
Yeah, I wouldn't try to add the ddev URL too -- too big a PITA.  I think you're right about splitting it into separate calls.

Reply:
okay great! thank you for the feedback!